Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs

```diff
@@ -260,15 +260,16 @@ public static IServiceCollection AddJellyfinApiSwagger(this IServiceCollection s
         {
             return serviceCollection.AddSwaggerGen(c =>
             {
+                var version = typeof(ApplicationHost).Assembly.GetName().Version?.ToString();
```

Comment:
Should this not default to something like 1.0.0? If it can really be null that is.

Reply:
I’m sure it’s nullable in case the version isn’t set, but it doesn’t hurt to have a default

Reply:
I'm just wondering if npm will complain if it's null